source/data/user_access.rst

```diff
@@ -0,0 +1,22 @@
+.. _user_access:
+
+User Access to iRODS
+====================
+
+Since Tier-1 Data Service platform is currently on the pilot phase, users of the system are consisted of volunteers. Thus, if you are invited or have an interest in, please contact with FOZ team.
+
+To be able to log on and to use Tier-1 Data Service platform, you need to have an iRODS account. Make sure that before claiming an iRODS account, a VSC account should be acquired. However not all VSC users might get an iRODS account. To this end you have to have a convenient project. 
+
+After the above mentioned points are assured, you can ask for an iRODS account via an email to FOZ.
+
+As soon as your user account is created by iRODS admin you will be notified. Keep in mind that your iRODS account will be exactly same as your VSC account.
+
+To reach out the iRODS you can use any login nodes of HPC system. As known well, when you log on to VSC system, the directory where you arrive by default is “VSC_HOME” that has a limited capacity, only for the typical configuration files etc. Therefore it is strongly advised to operate iRODS system in data directory which is pointed by the environment variable “VSC_DATA”. So first go to the “VSC_DATA” directory. 
```

Comment:
I'll mention HOME, DATA and SCRATCH later on but just to mention that they can transfer to iRODS data contained on any of those directories. 
```suggestion
Users can connect to the iRODS platform using different clients (command line, WebDAV interface or a portal) both from the VSC HPC systems (login and compute nodes) and from external systems (i.e: their labtops). 
```